core/src/wallet/bitcoin/explorers/LedgerApiBitcoinLikeBlockchainExplorer.cpp

```diff
@@ -46,12 +46,17 @@ namespace ledger {
             _explorerVersion = configuration->getString(api::Configuration::BLOCKCHAIN_EXPLORER_VERSION).value_or("v2");
         }
 
-        Future<String> LedgerApiBitcoinLikeBlockchainExplorer::pushLedgerApiTransaction(const std::vector<uint8_t> &transaction) {
+        Future<String> LedgerApiBitcoinLikeBlockchainExplorer::pushLedgerApiTransaction(const std::vector<uint8_t> &transaction, const std::string& correlationId) {
             std::stringstream body;
             body << "{" << "\"tx\":" << '"' << hex::toString(transaction) << '"' << "}";
             auto bodyString = body.str();
+            std::unordered_map<std::string, std::string> headers;
+            if(!correlationId.empty()) {
+                headers["correlationId"] = correlationId;
```

Comment:
The header should be named `ledger.correlation_id` according to https://ledgerhq.atlassian.net/wiki/spaces/VAUL/pages/2886074741/Logging+Guidelines#Normalising-the-data